platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -119,11 +108,9 @@
     @Mock
```

Comment:
We should make `configurationFileExtension` `static final` and replace every `+ ".config"` with it.